doc/semantics.md

```diff
@@ -80,6 +80,20 @@ if strict-floats are enabled, or
 
 otherwise.
 
+### `getClass()` for values of primitive types
```

Comment:
having "primitive types" in the title seems redundant. The entire section is about primitive data types.